net/crowdsec/Makefile

```diff
@@ -24,25 +22,24 @@ PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
 
 CWD_SYSTEM:=openwrt
-CWD_BUILD_VERSION?=$(PKG_SOURCE_VERSION)
+
+CWD_BUILD_VERSION?=v$(PKG_VERSION)
 CWD_BUILD_GOVERSION:=$(shell go version | cut -d " " -f3 | sed -E 's/[go]+//g')
 CWD_BUILD_CODENAME:=alphaga
-CWD_BUILD_TIMESTAMP:=$(SOURCE_DATE_EPOCH)
-CWD_BUILD_TAG:=openwrt
+CWD_BUILD_TIMESTAMP:=$(shell date +%F"_"%T)
```

Comment:
You should use `$(SOURCE_DATE_EPOCH)` even when you want to use a different date format (i.e. pass it into `date`) for reproducibility (to have the exact same binary executable every time you compile the same package with the same buildroot/sdk).

Reply:
@jefferyto 
Thanks, it makes sense !

I applied to https://github.com/openwrt/packages/pull/17989
on https://github.com/openwrt/packages/pull/17989/commits/3d451298593fe9270564e02ed74c46ede4b0f732

Reply:
I applied the same to https://github.com/openwrt/packages/pull/18015 on https://github.com/openwrt/packages/pull/18015/commits/d54d2319a4602f6e34e3aaa0bb6d634b9c21f65a